examples/bri-3/src/bri/workgroup/worksteps/api/worksteps.controller.ts

```diff
@@ -71,6 +73,17 @@ export class WorkstepController {
     );
   }
 
+  @Put('/:id/circuitinputsschema')
+  @CheckAuthz({ action: 'update', type: 'Workstep' })
+  async setCircuitInputsSchemaCommand(
```

Comment:
small thing but i would name this `update` instead of `set` to be clear, but up to you